src/game.cpp

```diff
@@ -53,4 +67,9 @@ void Game::spawn_wave() {
 	}
 	m_enemies.insert(m_enemies.end(), std::make_move_iterator(new_wave.begin()), std::make_move_iterator(new_wave.end()));
 }
+
+void Game::increase_score(std::size_t points) {
+	m_score += points;
+	m_score_text.set_string(m_font, "Score: " + std::to_string(m_score));
```

Comment:
Store a member string, clear and update via `std::format_to(std::back_inserter(string))` when score changes, then just pass that here.